internal_logs/internallog.txt

```diff
@@ -2126,3 +2126,273 @@
 2022-10-06 17:12:02.8123 Debug FileTarget(Name=logfile): Preparing for new file 'C:/Users/RALUCA/source/repos/AccountOwnerServer1/logs/2022-10-06_logfile.txt'
```

Comment:
Let's exclude this file from git. 
For ignoring files you can check this documentation: https://git-scm.com/docs/gitignore